ReactiveObjCTests/RACMulticastConnectionSpec.m

```diff
@@ -45,15 +45,16 @@
 		expect(@(subscriptionCount)).to(equal(@1));
 	});
 
-	qck_it(@"shouldn't reconnect after disposal", ^{
```

Comment:
The fact that there's a test for this means that the behavior is intentional.

I'm not entirely sure why this is the case. Maybe one of the other maintainers does. But I'm not personally comfortable accepting this PR given how little I know about it.